Formula/mingw-w64.rb

```diff
@@ -118,8 +119,17 @@ def install
 
       mkdir "mingw-w64-crt/build-#{arch}" do
         system "../configure", *args
-        system "make"
-        system "make", "install"
+        # Resolves "Too many open files in system"
+        # :info:build i686-w64-mingw32-dlltool: bfd_open failed open stub file
+        # dfxvs01181.o: Too many open files in system
+        # :info:build i686-w64-mingw32-dlltool: i686-w64-mingw32-dlltool: bfd_open
+        # failed open stub file: dvxvs00563.o: Too many open files in systembfd_open
```

Comment:
```suggestion
        # bfd_open failed open stub file dfxvs01181.o: Too many open files in system
        # bfd_open failed open stub file: dvxvs00563.o: Too many open files in systembfd_open
```
This should be enough information to identify the errors, I think.